File: src/backend/app/Models/Person.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\HasMany;

class Person extends Model
{
    use HasFactory;

    protected $table = 'person';

    protected $primaryKey = 'personId';
    
    public $timestamps = false;

    protected $fillable = [
        'userName',
        'firstName',
        'lastName',
        'emailAddress'
    ];

    public function user()
    {
        return $this->belongsTo(User::class);
    }

    public function phoneNumbers(): HasMany
    {
        return $this->hasMany(PhoneNumber::class, 'personId');
    }
}

File: src/asp_net_backend/src/api/Models/Person.cs
namespace Restify.API.Models;

public class Person
{
	public uint		Id { get; set; }
	public string	UserName { get; set; }
	public string	FirstName { get; set; }
	public string	LastName { get; set; }
	public string	EmailAddress { get; set; }

	public IList<PersonPhoneNumber>	PhoneNumbers { get; }
	public User	User { get; }

	public Person()
	{
		PhoneNumbers = new List<PersonPhoneNumber>();
	}
}